release_docs/README_HDF5_CMake

```diff
@@ -6,17 +6,18 @@ This tar file contains
     CTestScript.cmake
     HDF5config.cmake    CMake scripts for building HDF5
     HDF5options.cmake
-    hdf5-1.12.1         HDF5 1.12.1 source
+    hdf5-1.12.2         HDF5 1.12.1 source
```

Comment:
I'm confused with all these 1.12.2, 1.12.6, 1.12.7,...

Reply:
Fixed

Reply:
this is 1.12 dev branch therefore the source file would be the next release, 1.12.2

Reply:
OK then.